pkg/document/document.go

```diff
@@ -129,6 +129,11 @@ func (d *Document) ApplyChangePack(pack *change.Pack) error {
 	return nil
 }
 
+// InternalDocument returns the internal document.
+func (d *Document) InternalDocument() *InternalDocument {
+	return d.doc
+}
```

Comment:
This function is only used in the `store and find snapshots test` in `yorkie/backend/db/memory/db_test.go`.
Is there any other purpose for adding this function other than using it in tests?

If not, I think it's better to use `InternalDocument` by using `document.NewInternalDocument` in tests rather than exposing internal documents to the outside for testing. What do you think?

Reply:
It would be good to check this issue. I think you gave the same opinion as #125

Like `document.GarbageCollect` or `document.Checkpoint`, it doesn't need to be exposed to the user, but it's defined exposed because it makes it easy to write tests.

Exposing internal methods to some users leaves room for misuse. However, I think that even if internal methods are exposed to the user, most users can distinguish the necessary situation and use it.

And We haven't found a good way to control this as easily as the JS SDK(https://github.com/yorkie-team/yorkie-js-sdk/issues/181). I am concerned that the structure of the program becomes complicated or the development productivity decrease by trying to control exposed methods forcibly.

So I think it's ok to proceed as it is for now until we find a good way like the JS SDK.